cmd/panfs.go

```diff
@@ -519,15 +525,19 @@ func (fs *PANFSObjects) GetBucketInfo(ctx context.Context, bucket string, opts B
 	}
 
 	createdTime := st.ModTime()
-	meta, err := globalBucketMetadataSys.Get(bucket)
+	meta, err := loadBucketMetadata(ctx, fs, bucket)
+	// meta, err := globalBucketMetadataSys.Get(bucket)
```

Comment:
Please, don't leave commented-out code without a good reason (e.g. an example of usage of a function).